src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -123,27 +123,24 @@ If in doubt, our recommendation would generally be to go with Substantial (`urn:
 
 ## MitID for business ('MitID Erhverv')
 
-You can find the details regarding MitID Erhverv [here](/verify/e-ids/danish-mitid-erhverv/). 
+You can find the details regarding MitID Erhverv [here](/verify/e-ids/danish-mitid-erhverv/).
 
 ## Order MitID for production
 
 <Highlight icon="file-lines">
 
-  **Prerequisites for ordering**
+**Prerequisites for ordering**
 
-  In order to apply for MitID in production on behalf of a company you must meet the basic requirements:
+In order to apply for MitID in production on behalf of a company you must meet the basic requirements:
 
-  - Your company must be registered in the EU and have an EU VAT Id.
-  - The person applying must have a NemID/MitID employee signature from the company in question, a socalled "medarbejdersignatur"
-  - You must have completed step 5 in the [Getting ready for production](/verify/guides/production) guide. You will need the production domain to complete the order for your client credentials.
+- Your company must be registered in the EU and have an EU VAT Id.
+- The person applying must have a MitID employee signature from the company in question, a socalled "medarbejdersignatur"
```

Comment:
-> The person applying must be enrolled in MitID Erhverv for your company, or be a signatory of the company.